src/Validator/Rules/KnownDirectives.php

```diff
@@ -59,7 +61,9 @@ public static function unknownDirectiveMessage($directiveName)
     }
 
     /**
-     * @param (Node|NodeList)[] $ancestors
+     * @param Node[]|NodeList[] $ancestors
```

Comment:
Previous version was correct

![image](https://user-images.githubusercontent.com/327717/44724979-8969c880-aad4-11e8-9eb7-9bc937dbee77.png)

Reply:
But this version is correct too, I'll change if this can make the code great again :laughing:

Reply:
done!

Reply:
It's seam that this syntax introduce error phpcs https://travis-ci.org/webonyx/graphql-php/jobs/423635233 can you please tell me what is going wrong here?

Reply:
can you please provide a example I don't really get how to note an annotation sorry